ui/src/components/system/Dashboard.vue

```diff
@@ -1761,4 +1853,7 @@ export default {
 </script>
 
 <style>
+  .raidError {
+    color: #470000;
```

Comment:
change to `#cc0000`, but I think there is a `.red` CSS class inside the cockpit-core project that you can use

Reply:
Used red class.